packages/side-sheet/src/index.tsx

```diff
@@ -40,7 +41,10 @@ const S = {
     display: flex;
     flex-direction: column;
     transform: ${_ => getTransform(_.position)};
-    ${_ => getBoxPosition(_.position)};
+    ${_ => ({
+      width: ['left', 'right'] ? getWidthType(_.width) : undefined,
```

Comment:
Actually this line makes little sense - `['left', 'right']` will always evaluate to `true`, hence this ternary operator will always result in truthy path. Please resolve this.